Google.PowerShell/Common/ActiveUserToken.cs

```diff
@@ -0,0 +1,74 @@
+﻿using Google.Apis.Util;
+using System;
+
+namespace Google.PowerShell.Common
+{
+    /// <summary>
+    /// OAuth 2.0 model for a successful access token response as specified in 
+    /// http://tools.ietf.org/html/rfc6749#section-5.1.
+    /// </summary>
+    public class ActiveUserToken
```

Comment:
Can we make this class immutable? e.g. converting all properties into `{ get; protected set; }`. The idea of modifying the fields of this seems super sketchy. Perhaps we should just have a constructor that parses the values out of a Json object?

e.g. `new ActiveUserToken(userTokenJson)`?